mmv1/templates/terraform/examples/oracledatabase_autonomous_database_basic.tf.tmpl

```diff
@@ -0,0 +1,22 @@
+resource "google_oracle_database_autonomous_database" "{{$.PrimaryResourceId}}"{
+  display_name = "autonomousDatabase displayname"
+  autonomous_database_id = "my-adb-instance"
+  location = "us-east4"
+  project = "{{index $.Vars "project"}}"
+  database = "testADBdb"
+  admin_password = "123Abpassword"
+  network = data.google_compute_network.default.id
+  cidr = "10.5.0.0/24"
+  properties {
+    compute_count = "2"
+    data_storage_size_tb="1"
+    db_version = "19c"
+    db_workload = "OLTP"
+    is_auto_scaling_enabled= "true"
+    license_type = "LICENSE_INCLUDED"
+    }
+}
+
+data "google_compute_network" "default" {
```

Comment:
I assume the network needs to live in the same project as the resource; we'll want to specify `project` on this. If we want to create the project per test run we'll want to use the resource instead of the datasource (https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_network). If there's an already-created network we should be attaching to, we'd reference it here in the datasource.

Reply:
Thanks added project field in network datasource worked.